README.md

```diff
@@ -11,6 +11,45 @@ We'd love your help!. Use tags [up-for-grabs][up-for-grabs-issues] and
 
 We have a weekly SIG meeting! See the [community page](https://github.com/open-telemetry/community#javascript-sdk) for meeting details and notes.
 
+## Packages
+
+### API
+
+| Package                 | Description |
+| ----------------------- | -----------------|
+| [Types](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-types) | This package provides TypeScript interfaces and enums for the OpenTelemetry core trace and metrics model. It is intended for use both on the server and in the browser. |
```

Comment:
Would it make sense to give the full package names, i.e. making it be `[@opentelemetry/types](...) | ` and similar below? People may think in terms of the full names since they will need them to NPM/Yarn install them.

Reply:
sure, done.